<launch>

    <arg name="node_namespace" default="apriltag" />

    <!-- Set parameters -->
    <rosparam command="load" file="$(find cv_arm_control)/config/tag_settings.yaml" ns="$(arg node_namespace)" />
    <rosparam command="load" file="$(find cv_arm_control)/config/tags.yaml" ns="$(arg node_namespace)" />

    <node pkg="apriltag_ros" type="apriltag_ros_single_image_server_node" name="$(arg node_namespace)" clear_params="true" output="screen" />

</launch>